lighthouse-core/config/default.json

```diff
@@ -387,11 +412,11 @@
           "weight": 1
         },
         "critical-request-chains": {
-          "expectedValue": 0,
```

Comment:
The `rawValue` that we display is numeric, yes. But we treat the audit as a pass/fail, which means `expectedValue` needs to be a boolean. AFAICT, `expectedValue` should really be `expectedType`.